easybuild/easyconfigs/m/magma/magma-1.6.2-CrayGNU-5.2.40.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = "ConfigureMake"
+
+name = 'magma'
+version = '1.6.2'
+
+homepage = 'http://icl.cs.utk.edu/magma/'
+description = "The MAGMA project aims to develop a dense linear algebra library similar to LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems."
+
+toolchain = {'name': 'CrayGNU', 'version': '5.2.40'}
+toolchainopts = {'pic': True, 'dynamic': True,  'openmp': True, }
+
+preinstallopts = "export EBINSTALLPREFIX=%(installdir)s && "
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://icl.cs.utk.edu/projectsfiles/magma/downloads/']
+
+skipsteps = ['configure']
+
+builddependencies = [ ('cudatoolkit/6.5.14-1.0502.9613.6.1', EXTERNAL_MODULE) ]
```

Comment:
This is provided by Cray (version numbers might change), but we need to provide the external modules metadata:
[cudatoolkit/6.5.14-1.0502.9613.6.1]
name = CUDA
version = 6.5.14-1.0502.9613.6.1
prefix = CRAY_CUDATOOLKIT_DIR

We discussed with @pforai and @boegel, it would be nice to have a standard file cray_external_modules_metadata.cfg shipped with EB.